### ISSUE TITLE
Excessive transform logs in Gradle 4.9 console

### ISSUE THREAD

**Issue by hungvietnguyen** [gradle/gradle#6092]
### Environment
Gradle 4.9 (no issues with earlier versions)

### Current Behavior
When building a small project ([MyApplication.zip](https://github.com/gradle/gradle/files/2229984/MyApplication.zip)) with `./gradlew clean assembleDebug --console=plain`, Gradle outputs multiple lines of the following form, which clutters the console:

```
> Transform constraintlayout-1.1.2.aar with ExtractAarTransform
> Transform appcompat-1.0.0-alpha1.aar with ExtractAarTransform
> Transform kotlin-stdlib-jdk7.jar (org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.2.50) with JetifyTransform
> Transform c2f0571aaabbdfff9ca547969ec4a743 with AarTransform
> Transform fa142b411c8f14084d00b43b2ba21dc7 with AarTransform
> Transform kotlin-stdlib-jdk7-1.2.50.jar with IdentityTransform
```

This happens even with incremental, no-op builds.

### Expected Behavior
Transform logs should be output with `--info` only.

### Context
This issue was originally reported at http://issuetracker.google.com/111626453

**Comment by lptr** [comment]
Thanks @hungvietnguyen, looking into this for 4.10.

**Comment by lptr** [comment]
The situation should be much better in 4.10. We'll have further improvements in the next Gradle versions. @hungvietnguyen please reopen if you still see problematic output with the latest nightly.

**Comment by SUPERCILEX** [comment]
@lptr I was the original issue reporter (on the Google side), but this isn't fixed in 4.10-rc-1. I'm still getting spammed:
![image](https://user-images.githubusercontent.com/9490724/43972355-eab7fb20-9c88-11e8-86d2-c92b718c7934.png)

**Comment by SUPERCILEX** [comment]
@lptr Actually, it has improved because there are no longer thousands of lines of transforms. But still, there are a few left.
